app/models/course_instance.rb

```diff
@@ -18,6 +18,10 @@ class CourseInstance < ActiveRecord::Base
     .unenrolled(user)
   }
 
+  def schedule
+    meetings.map(&:schedule).join('; ') || 'TBA'
```

Comment:
Ah damn. I'll go back to the original. I can still do it in one but I don't think it's as clear: `(meetings.map(&:schedule) || ['TBA']).join('; ')`